server/util/flagutil/flagutil.go

```diff
@@ -240,3 +269,176 @@ func (f *URLFlag) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	_, err = url.Parse(string(*f))
 	return err
 }
+
+// Generates a map of the type that should be marshaled from YAML for each flag
+// name at the corresponding nested map index.
+func GenerateYAMLTypeMapFromFlags() (map[string]interface{}, error) {
```

Comment:
In go, the official way to write comments is to start them with the function name. So this would be:

// GenerateYAMLTypeMapFromFlags generates a map of the type that should be
// marshaled from YAML for each flag name at the corresponding nested map index.


(can you double check the rest of the comments in here too?)

Reply:
Thanks for the info! Done.